lib/api.js

```diff
@@ -24,10 +23,21 @@ var certFile = path.resolve(__dirname, '../config/cert.pem');
 // Start express server
 var express = require('express');
 var app = express();
-var server = https.createServer({
-  key: fs.readFileSync(keyFile),
-  cert: fs.readFileSync(certFile),
-}, app);
+var server;
+
+// We default to https protocol, if certs are not found fallback to http
+var  protocol = 'https';
+try {
+  server = require('https').createServer({
+    key: fs.readFileSync(keyFile),
+    cert: fs.readFileSync(certFile),
+  }, app);
+} catch (e) {
+  server = require('http').createServer(app);
+  protocol = 'http';
+
+}
```

Comment:
Still going to use try/catch, but only checking for ENOENT from the `certFile`. The alternative is to use the asynchronous version, but a decent chunk of the code below would have to go in the callback.